.github/labeler.yml

```diff
@@ -1,16 +1,40 @@
 # SPDX-License-Identifier: Apache-2.0
 
+CI/CD:
+  - changed-file:
+      - any-glob-to-any-file:
+          - .github/labeler.yml
```

Comment:
How about `.github/*`

Reply:
`PULL_REQUEST_TEMPLATE.md` shouldn't be included IMO